src/modules/form.js

```diff
@@ -0,0 +1,42 @@
+import User from './userClass.js';
+
+window.addEventListener('DOMContentLoaded', () => {
+// Dom Selectors
+  const form = document.querySelector('#form');
+  const userName = document.querySelector('#name');
+  const userScore = document.querySelector('#scores');
+  const refresh = document.querySelector('#refresh');
+  const scoresData = document.querySelector('.scores-list');
+
+  // Clear userInput
+  const clearInput = () => {
+    userName.value = '';
+    userScore.value = '';
+  };
+
+  // Get userData
+  const getUserData = (e) => {
+    const user = new User(userName.value, userScore.value);
+    user.createUserScore(user);
+    clearInput();
+    e.preventDefault();
+  };
+
+  // Get scores
+  const getScores = async () => {
+    let list = '';
+    const scores = await User.fetchScores();
+    const scoresList = scores.result;
+    scoresList.map((scorelist) => {
+      list += `<li><span>${scorelist.user}:</span> ${scorelist.score}</li>`;
+      return list;
+    });
+    scoresData.innerHTML = list;
+  };
+
+  // Refresh to get scores from api
+  refresh.addEventListener('click', getScores);
+
+  // Get form input
+  form.addEventListener('submit', getUserData);
```

Comment:
**OPTIONAL** - Good job done so far, though your code works, but to make your project more readable and easier to maintain, I suggest that you do not need to put all these lines of code in the `DOMContentLoaded` event listener, Please place them outside the event listener 👍 👍 

**REQUIRED** - When your page loads initially, please fetch all scores and display them on the UI, currently I have to click on the refresh button to fetch the scores on the initial render. Please fix this 🙏 
Your project:
<img width="960" alt="nrftnrtn" src="https://user-images.githubusercontent.com/73607512/223996724-6f27df24-63fb-4933-a437-03f615f24510.PNG">